frontend/src/app/plot/plot.component.ts

```diff
@@ -44,35 +44,61 @@ export class PlotComponent implements OnInit {
    *      for each datapoint. 'lines' displays a line through all datapoints. 'lines+markers'
    *      displays both.
    */
-  plot_data = [ {x: [0], y: [0], type: 'scattergl', mode: 'markers'}]
-  message: any
-  constructor(private sharedService: UploadService) { }
+  plot_data = [ {x: [0], y: [0], type: 'scattergl',
+   mode: 'markers', id: 0, visible: 'true',
+   name: 'Placeholder Point'
+  }]
 
-  ngOnInit(): void {
+  // Plot Configurations.
+  plot_layout = { title: 'Add a new dataset.', legend: 'false' }
+  plot_config = { scrollZoom: true, displayModeBar: true}
+  
+  message: any
+  constructor (private sharedService: UploadService) { }
 
+  ngOnInit (): void {
```

Comment:
I'm not sure what "ng" means. Can you document this? Maybe fix the naming if its not obvious?